src/EmailCardList/EmailCardList.js

```diff
@@ -0,0 +1,22 @@
+import EmailCard from "./EmailCard/EmailCard";
+
+const EmailCardList = (props) => {
+    const formatDateForDisplay = (emailDate) => {
+        const headerDateUnix = Date.parse(emailDate)
+        const headerDate = new Date(headerDateUnix)
+        return headerDate.toLocaleDateString("en-GB")
```

Comment:
here the name `headerDate` is quite mysterious. i would suggest renaming the consts to `dateUnixTimestamp` and `dateObject`